### DISCUSSION TITLE
Is there a way to set notifications location?

### DISCUSSION THREAD

**Discussion by drbraden** [Belphemur/SoundSwitch#981] [Q&A] [answered]
First off, I just discovered SoundSwitch today, loving the application!

Can the location of notifications be changed?  On my system they are currently displayed in the top left corner, but would much prefer they appear in the top right corner.

**Comment by Belphemur** [comment, marked as answer]
Hello,

There isn't a way to change the location of the banner for now.
There is an issue about this; #242
